### ISSUE TITLE
Train problems

### ISSUE THREAD

**Issue by SaintLogos1234** [bilylee/SiamFC-TensorFlow#66]
For personal needs, I used my own dataset to run the siamfc framework, because my dataset was relatively small, and this warning appeared during the training：
2018-12-11 09:55:17.278447: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:18.115279: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:18.486890: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:18.658596: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:18.737470: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
INFO - root - 2018-12-11 09:55:18.943072: step 332510, total loss = 0.91, batch loss = 0.84 (78.8 examples/sec; 0.102 sec/batch; 3h:45m:00s remains)
2018-12-11 09:55:19.037643: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:19.252102: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:19.566876: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:19.773020: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
INFO - root - 2018-12-11 09:55:19.993944: step 332520, total loss = 0.60, batch loss = 0.54 (79.5 examples/sec; 0.101 sec/batch; 3h:43m:09s remains)
2018-12-11 09:55:20.090338: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:20.318091: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:20.663695: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
2018-12-11 09:55:20.816399: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
INFO - root - 2018-12-11 09:55:21.111168: step 332530, total loss = 0.76, batch loss = 0.69 (58.0 examples/sec; 0.138 sec/batch; 5h:05m:38s remains)
2018-12-11 09:55:21.181763: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
^C2018-12-11 09:55:21.232668: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: exceptions.StopIteration: Iteration finished.
It can be trained, I would like to ask, this will affect the training results?

**Comment by bilylee** [comment]
This warning just means the dataloader has reached the end of one epoch. It won't affect training since it will reiterate from the start. However, with small training set, you may face the overfitting issue.

**Comment by SaintLogos1234** [comment]
Thank you very much for your reply
